dodo/search.py

```diff
@@ -201,6 +207,7 @@ def __init__(self, a: app.Dodo, q: str, keep_open: bool=False, parent: Optional[
         self.setStyleSheet(f'QTreeView::item {{ padding: {settings.search_view_padding}px }}')
         self.model = SearchModel(q)
         self.tree.setModel(self.model)
+        self.model.modelReset.connect(lambda: self.has_refreshed.emit())
```

Comment:
```suggestion
        self.model.modelReset.connect(self.has_refreshed)
```

(you should be able to directly connect signals to other signals)

Reply:
Thanks for the hint. I ended up with a proper method since I folded it with some better error reporting.